content/projects/singlecell/index.md

```diff
@@ -0,0 +1,77 @@
+---
+description: "Users, developers, and trainers providing feedback and energy to improve and expand the Galaxy single cell capabilities."
+autotoc: true
+title: "Galaxy Single Cell Community of Practice"
+---
+
+We are a mix of users, developers, and trainers providing feedback and energy to improve and expand the Galaxy single cell capabilities. Do you use Galaxy single cell tools for scRNA-seq or beyond? Do you have feedback for us? Want to be a tester of new resources? Want to impact the future of single cell in Galaxy? Are you a developer looking for a place to play? Are you developing Galaxy single cell materials and want to stay aware of what others are doing in this field?
+
+**Welcome to the Galaxy Single-Cell Community of Practice!**
+
+## I'm just trying to learn to analyse my data, how did I get here? 
+Hello user! You'll be interested in the following resources:
+
+ - [Learning pathway for getting to grips with Galaxy and single cell](https://training.galaxyproject.org/training-material/learning-pathways/intro_single_cell.html)
+ - [Single cell training material](https://training.galaxyproject.org/training-material/topics/single-cell/)
+ - Galaxy instance: [Single cell instance](https://singlecell.usegalaxy.eu) or Human Cell Atlas instance](https://humancellatlas.usegalaxy.eu)
+ - [Matrix channel for asking for help](https://matrix.to/#/#Galaxy-Training-Network_galaxy-single-cell:gitter.im)
+
+Saying that, we'd absolutely love to have users feedback on their experiences to help us improve. You'd be very welcome to join the Community of Practice to help improve the Galaxy single cell space!
+
+# What are the goals of the single cell community of practice?
+
+🎯Goals
+- Prevent work duplication
+- Collaborate across pipeline, ensuring users feedback to developers
+- Identify community needs
```

Comment:
Benchmarking tools and workflows?

Reply:
Interesting! Do we benchmark them? Benchmarking assumes testing quality of them against each other... so like deciding which method is best for analysis. Is this what you meant? Or something else?

Reply:
I thought it is one of the goals of the upcoming staff exchange program.

Reply:
Added!